.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug to help us improve.
```

Comment:
We should add default labels as well. freud should be better about labeling issues in general if possible. This one should be 'bug'.